packages/components/src/Avatar/Avatar.mdx

```diff
@@ -112,3 +112,6 @@ If an avatar is displayed without a name label, display a tooltip.
     );
   }}
 </Playground>
+
+## Figma
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FIWkxs8u2ltcUM2lGv48SHf%2FAtlantis-Components-Public%3Fnode-id%3D2%253A9&chrome=DOCUMENTATION" allowfullscreen></iframe>
```

Comment:
Setting `width="100%"` will allow this to flex to whatever width the main column is